infrastructure/main.tf

```diff
@@ -6,5 +6,6 @@ locals {
       "Destroy Me"   = var.destroy_me
     })
   ))
+  civil_service_resource_group = "${var.product}-service-${var.env}"
   civil_shared_resource_group = "${var.product}-shared-${var.env}"
```

Comment:
please remove this

Reply:
Removed both resource group local variables as now using a new civil-sdt resource group which is defined in this file.